svf/lib/SVFIR/SVFVariables.cpp

```diff
@@ -178,6 +178,16 @@ const std::string ArgValVar::toString() const
     return rawstr.str();
 }
 
+GepValVar::GepValVar(ValVar* baseNode, const SVFValue* val, NodeID i,
+                     const AccessPath& ap, const SVFType* ty)
+    : ValVar(val, i, GepValNode), ap(ap), base(baseNode), gepValType(ty)
+{
+    if(baseNode->ptrInUncalledFunction())
```

Comment:
Do we need to have these two set methods?

Just add a method in GepValVar: "isPtrInUncalledFunction() { return baseNode->ptrInUncalledFunction()}"